CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Change Log
+# [v5.2.4](https://github.com/marklogic/marklogic-data-hub/releases/tag/v5.2.4) (2020-09-03)
+- Note: There are no changes from v5.2.3
```

Comment:
I think we need to say something - this doesn't really make sense otherwise. Even if it's just "Fix in Explorer app".